sdp-utils.c

```diff
@@ -1371,26 +1374,29 @@ janus_sdp *janus_sdp_generate_answer(janus_sdp *offer, ...) {
 				}
```

Comment:
Shouldn't the two above be part of the `if(codec_rtpmap)` too, especially `audio_fmtp` since it relies on a payload type that may not be associated to a codec at all? I see you did that for video.

Reply:
I wasn't sure for the DTMF (could you have an invalid/disabled audio codec but an independent  DTMF line)? You're definitely right about the fmtp though.